sdk-setup/src/mb2

```diff
@@ -2070,6 +2075,24 @@ run_package() {
             -bb --noprep --nobuildstage --noclean \
             "$TMP_SPEC"
     )
+    [[ -z $NO_SIGN_RPM ]] && run_sign
+}
+
+run_sign() {
+    [[ -e $STATEDIR/spec ]] || fatal "No previoud build found"
+    local retcode=0
+    rpms=$(find_rpms $all_opt "$OPT_OUTPUTDIR" "$STATEDIR/spec")
+    [[ $rpms ]] || fatal "No package found"
+    local sdksign=/usr/bin/sdk-rpmsign
+
+    if [[ ! -f $sdk-sign ]]; then
+        rpmsign --addsign ${rpms}
+    else
+        $sdk-sign $@ ${rpms}
```

Comment:
As discussed before either the arguments should be the same in both cases. If we need to support signing tools with different option set, we need to provide more appropriate UI for it.

What would be an example of options not recognized by standard rpmsign?

Reply:
If use custom sign tools it may be additional signature keys or parameters